### DISCUSSION TITLE
The OSPO Wishlist

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#174] [OSPO Wishlist]
Hello OSPOers!

TODO is working to create and nurture bi-directional communication between [open source communities that are helping the OSPO movement](https://todogroup.org/associates/) on different areas like:

* Open Source Security
* Supply chain standards
* OSPOs adoption across industries
* Etc

In order to gather community feedback and allow these communities to better assist OSPO inquiries, you can now leave your requests under this new [OSPO wishlist category](https://github.com/todogroup/ospology/discussions/categories/ospo-wishlist) 💜

**Comment by CsatariGergely** [comment]
Let's dream big. Here is my wishlist:

- Clearly declare the license of your project. If your licensing is difficult use https://reuse.software/
- If you provide any binary delivery including container images attach an SBOM to it
- Do not freak out if contributors add themselves to the copyright notices

**Comment by fbim-genosr** [comment]
It would be nice to address things like training for OSPOs in a governmental context.
